nservicebus/sqlserver/multiple-databases.md

```diff
@@ -14,24 +14,48 @@ The SQL Server transport allows you to select, on per-endpoint basis, where the
 
 The transport will route messages to destination endpoints based on the configuration. If no specific configuration has been provided for a particular destination endpoint, the transport assumes the destination has the same configuration (schema, database and instance name/address) as the sending endpoint. If this assumption turns out to be false (the transport cannot connect to destination queue), an exception is thrown immediately. There is no store-and-forward mechanism on the transport level (and hence -- no dead-letter queue).
 
-NOTE: If the destination endpoint uses different database or server instance, sending a message to it might cause the transaction to escalate to a distributed transaction. Usually it is not a desired effect and one can use NServiceBus Outbox to avoid it.
+NOTE: If the destination endpoint uses different database or server instance, sending a message to it might cause the transaction to escalate to a distributed transaction. Usually it is not a desired effect. Use NServiceBus [Outbox](/nservicebus/outbox/) to avoid it.
```

Comment:
@mikeminutillo - Outbox has constraints. It needs to be in the same database as that of the business db. See http://docs.particular.net/nservicebus/outbox/#caveats.
What I am not sure is what happens when the outbox is trying to send it to different queues located in different databases. Is this scenario documented? For example, what happens if the handler transaction is ok but sending to one of these database didn't succeed due to datbase connection issues? @SzymonPobiega @mauroservienti ? But these are things that should in the outbox doco. 

@weralabaj - minor corrections on your note. I think we should call out the caveats as well. See below:

NOTE: If the destination endpoint uses a different database or server instance, sending a message to it might cause the transaction to escalate to a distributed transaction which may not be desirable. Using the [Outbox](/nservicebus/outbox/) feature, DTC escalations can be avoided as long as the both the endpoint's Outbox and Business data share the same database.